elbepack/makofiles/preferences.mako

```diff
@@ -25,16 +55,15 @@ Pin-Priority: ${porg['pin']}
 % endfor
 % if pkgs:
 %  for n in pkgs:
-%   if "pin" in n.et.attrib.keys():
-Package: ${n.et.text}
-Pin: release n=${n.et.attrib["pin"]}
-Pin-Priority: 991
-
-%   endif
-%   if "version" in n.et.attrib.keys():
+%   if pkg_properties_specified(n.et.attrib):
 Package: ${n.et.text}
+%    if pkg_origin_release_specified(n.et.attrib):
+${generate_origin_release_pin_text(n.et.attrib)}
+%    endif
+%    if "version" in n.et.attrib.keys():
 Pin: version ${n.et.attrib["version"]}
-Pin-Priority: 1001
+%    endif
+Pin-Priority: ${n.et.attrib.get("prio", 1001)}
```

Comment:
Using prio like this does not really match its documentation in the schema.

Do you need it for your usecase?
Looking at it now it doesn't seem very useful to make this configurable.

Reply:
(If it is to stay, it should move to its own commit)

Reply:
I'm fine with just taking 1001.

Reply:
Thanks, that's an easier interface to commit to long-term.